sdk/storage/azure-storage-datamovement/inc/azure/storage/datamovement/storage_transfer_manager.hpp

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include <string>
+
+#include <azure/storage/blobs.hpp>
+
+#include "azure/storage/datamovement/blob_folder.hpp"
+#include "azure/storage/datamovement/datamovement_options.hpp"
+#include "azure/storage/datamovement/job_properties.hpp"
+
+namespace Azure { namespace Storage { namespace DataMovement {
+
+  class StorageTransferManager final {
```

Comment:
Why is this a `Manager` and not a `Client`?  ,

Reply:
It's transfer manager in .Net